dev-docs/plugins/bc/bc-prebid-plugin-render-options.md

```diff
@@ -259,6 +261,67 @@ No
 
 `options.adText = 'Publicité'`
 
+<a name="wrapperLimit"></a>
+### wrapperLimit
+
+**Description:**
+
+Specifies the maximum number of XML redirects that are allowed to be considered when attempting to play an ad.
+
+**Acceptable Values:**
+
+Integer
+
+The number of XML strings being parsed in a wrapper chain to obtain the media file to play. This includes the number of wrapper XML strings and the inline XML.
```

Comment:
Change this line to :  The number of XML documents being parsed in a wrapper chain to obtain the media file to play. This includes the number of wrapper XML documents and the inline XML.

Reply:
fixed